.eslintrc.js

```diff
@@ -7,4 +7,24 @@ module.exports = {
       rootDir: ['apps/*/', 'packages/ui/*/', 'packages/onboard-helpers/*/'],
     },
   },
+  overrides: [{
+    files: [
+      'apps/{loan,lend}/src/{entities,features}/**/*.{ts,tsx}',
```

Comment:
Add all the FSD layers
![image](https://github.com/user-attachments/assets/bd33398d-b60d-4047-9aac-86e270e142f7)